package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "callhome-nodejs",
-  "version": "1.2.2",
+  "name": "split-evaluator",
+  "version": "1.3.0",
```

Comment:
@patricioe this version change ... means that `splitsoftware/split-evaluator:1.2.2` should be renamed on README file no?